Tools/adb_upload_to_bebop.sh

```diff
@@ -34,11 +34,25 @@ adb shell mount -o remount,rw /
 adb shell touch /home/root/parameters
 adb shell mkdir -p /data/ftp/internal_000/fs/microsd
 
-${RPI_TOOLCHAIN_DIR}/gcc-linaro-arm-linux-gnueabihf-raspbian/bin/arm-linux-gnueabihf-strip \
+# kill PX4 if it is already running from autostart
+restart_px4=false
+adb_return=$(adb shell killall -KILL px4)
+if [[ $adb_return == "" ]]; then
+    echo "Killed running PX4 process"
+    restart_px4=true
+fi
+
+`dirname ${PX4_CXX_COMPILER}`/arm-linux-gnueabihf-strip \
   -R .comment -R .gnu.version \
   ../build_posix_bebop_default/src/firmware/posix/px4
```

Comment:
Do this as a cmake custom command as it will already have the binary path and arm-linux-gnueabihf-strip.